packages/generated/scripts/make-config.sh

```diff
@@ -1,6 +1,8 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Not bash?

Reply:
couldn't get it to build on alpine without sh. but i think i need to rethink my workarounds here, because now CI is failing